.github/workflows/CI-unittests.yml

```diff
@@ -26,25 +26,25 @@ jobs:
             python-version: "3.9"
```

Comment:
I did not understand why we keep ubuntu-latest with mamba and macos-latest for version 3.9 if you decided not to support 3.9. I would expect either we support it completely or we don't support at all. What is the idea behind it?

Reply:
I took your comment in the discussions into account and decided to keep 3.9 as long as DPPS supports 3.9. I hope this is ok - we lower the number of tests by a x2, so this is progress.